Plugins/Encounter/NWScript/nwnx_encounter.nss

```diff
@@ -89,6 +89,22 @@ int NWNX_Encounter_GetMaxNumSpawned(object encounter);
 /// @return amount of creatures
 int NWNX_Encounter_GetCurrentNumSpawned(object encounter);
 
+/// @brief Get the geometry of an encounter
+/// @param oEncounter: The encounter object.
+/// @return A string of vertex positions.
+string NWNX_Encounter_GetEncounterGeometry(object oEncounter);
```

Comment:
Naming nit: Since this is already namespaced to `nwnx_encounter` there is no need to specify "Encounter" twice. Just `NWNX_Encounter_{G,S}etGeometry()` would do.